### ISSUE TITLE
Missing icons from ChonkyIcon.js

### ISSUE THREAD

**Issue by trivigy** [TimboKZ/Chonky#28]
Getting an error trying to load it. Would greatly appreciate some help figuring this out.

```
ERROR in ./node_modules/chonky/lib/components/external/ChonkyIcon.js
Module not found: Error: Can't resolve '@fortawesome/free-brands-svg-icons/faAdobe' in '/home/trivigy/Projects/trusty/node_modules/chonky/lib/components/external'                                                         
 @ ./node_modules/chonky/lib/components/external/ChonkyIcon.js 19:16-69                                                                                                                                                                 
 @ ./node_modules/chonky/lib/index.js                                                                                                                                                                                                   
 @ ./src/components/Panels/Files/Files.tsx                                                                                                                                                                                              
 @ ./src/components/Panels/Files/index.ts                                                                                                                                                                                               
 @ ./src/components/Panels/index.ts
```

**Comment by TimboKZ** [comment]
Thanks for reporting this and finding the root cause.

Quoting your message from Discord for visibility:
> [...] the issue is with latest fortawesone. The npm version is missing a few icons for some reason. Should not be used. 5.15.0.

Recommended solution for now would be to downgrade to an earlier version of Fortawesome - Chonky should work with any version after `5.13.x`.

I will update the Chonky's `package.json` to either blacklist `5.15` or to use stricter versions.

**Comment by TimboKZ** [comment]
I have just released [Chonky v1.1.3](https://github.com/TimboKZ/Chonky/releases/tag/v1.1.3), which should fix this issue.
